test/integration/controller/jobset_controller_test.go

```diff
@@ -79,13 +80,17 @@ var _ = ginkgo.Describe("JobSet validation", func() {
 			// If we are expected a validation error creating the jobset, end the test early.
 			if !tc.jobSetCreationShouldSucceed {
 				ginkgo.By("checking that jobset creation fails")
-				gomega.Expect(k8sClient.Create(ctx, js)).To(gomega.Not(gomega.Succeed()))
+				gomega.Eventually(func() error {
+					return k8sClient.Create(ctx, js)
+				}, timeout, interval).ShouldNot(gomega.Succeed())
```

Comment:
we can remove this logic and the field `jobSetCreationShouldSucceed`, there is only a single test case that doesn't use it.

Reply:
done.